backend/pkg/leader/config.go

```diff
@@ -14,7 +14,8 @@ func Config() (Configuration, error) {
 	// TODO(juliaogris): read from config file and/or command line
 	initialCursor := ""
 	createdAfter, _ := time.Parse(time.RFC3339, "2019-05-15T00:00:00Z")
-	labelPattern := `^lab`
+	labelPattern := `lab*`
```

Comment:
In the next commit, the test data uses `lab.*` instead of `lab*`. Is this meant to be a regex, or a glob? If the latter, can you call it labelGlob? Globs and Regexes are two different pattern matchers. Globs are the command line ones.

Reply:
oh - finally somebody shed light onto this messy corner of my brain.
Thank you! Done.